checkout/src/main/java/com/checkout/validation/validator/CVVComponentDetailsValidator.kt

```diff
@@ -0,0 +1,14 @@
+package com.checkout.validation.validator
+
+import com.checkout.base.model.CardScheme
+import com.checkout.validation.api.CVVComponentValidator
+import com.checkout.validation.model.CvvValidationRequest
+import com.checkout.validation.validator.contract.Validator
+
+internal class CVVComponentDetailsValidator(private val cvvValidator: Validator<CvvValidationRequest, Unit>) :
+    CVVComponentValidator {
+
+    override fun validateCvv(cvv: String, cardScheme: CardScheme) = cvvValidator.validate(
+        CvvValidationRequest(cvv, cardScheme)
+    )
+}
```

Comment:
Any reason why we need a factory layer for providing the single instance? 

Shall we expose and inject `CvvValidator` directly into the `CVVComponentViewModelFactory`, where it could be more efficient? and in this case we don 't need to create the wrapper `CVVComponentValidator` and  `CVVComponentValidatorFactory `.

Reply:
CVVValidator is the core implementation part of Cardvalidator and both are parts of the Checkout module. So, need to create a new public CVV validator for it.